### PR TITLE
HP-153 - Logged In Dropdown

### PR DESCRIPTION
**Pull request by Audiosutras**
https://caktus.atlassian.net/browse/HP-153
This PR:
1) Adds logged-in dropdown based off of this figma wireframe: https://www.figma.com/file/OZAfVKxh8fs8XE7RLnlbY7/PDPH-COVID19?node-id=1697%3A4313
2) Refactors header.html

User should be able to close the dropdown by clicking outside of the dropdown area.

### CODE REVIEW

File: apps/hip/static/js/loggedInDropdown.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const loggedInBtn = document.querySelector("#logged-in-dropdown-btn");
+  if (loggedInBtn) {
+    manageDropdownIsActiveClickEvent(loggedInBtn);
+  }
+  function manageDropdownIsActiveClickEvent (btn) {
+    const dropdown = document.querySelector("#logged-in-dropdown");
+  
+    btn.addEventListener("click", function () {
+      const dropdownIsActive = dropdown.classList.contains("is-active");
+      console.log(dropdownIsActive)
```
Comment (vkurup) [inline apps/hip/static/js/loggedInDropdown.js:11]: Leftover debug statement ^

File: hip/static/js/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ import Header from "../../../apps/hip/static/js/header";
 import SideBar from "../../../apps/hip/static/js/sidebar";
 import RightScrollNav from "../../../apps/hip/static/js/rightScrollNav";
 import MobileSearchModal from "../../../apps/hip/static/js/mobileSearchModal";
+import loggedInDropdown from "../../../apps/hip/static/js/loggedInDropdown";
```
Comment (vkurup) [inline hip/static/js/index.js:26]: Any reason this is not uppercased like the other components?

**Review by vkurup — APPROVED** [review]
Couple minor comments, but this looks and works great! :lock:

File: hip/static/js/index.js (hunk 2 above)
Comment (Audiosutras) [reply hip/static/js/index.js:26]: No, just zoned out and made it camel case.

File: apps/hip/static/js/loggedInDropdown.js (hunk 1 above)
Comment (Audiosutras) [reply apps/hip/static/js/loggedInDropdown.js:11]: thank you

**Pull request merged**